ensimesublime/protocol.py

```diff
@@ -238,7 +239,13 @@ def apply_refactor(self, call_id, payload):
             self.env.logger.warning("Couldn't parse diff_file: {}"
                                     .format(diff_file))
             return
-        result = patch_set.apply(0, "/")
+        path =  Path(self.refactorings[payload['procedureId']])
+        self.env.logger.debug("Refactoring get root from: {}"
+                                 .format(self.refactorings[payload['procedureId']]))
+        root = path.drive or path.root
+        self.env.logger.debug("Refactoring set root: {}"
+                                 .format(root))
+        result = patch_set.apply(0, root)
```

Comment:
You might want to pass str(root) to avoid `TypeError: argument should be string, bytes or integer, not PosixPath`